modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,10 +181,33 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+where:
+
+<num>:: Specifies the worker node number.
+
+. Inspect the bare-metal node by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get bmh openshift-worker-<num>
+----
++
+where:
 
+<num>:: Specifies the worker node number.
++
+.Example output
+[source,terminal]
+----
+NAME                    STATE       CONSUMER   ONLINE   ERROR
+openshift-worker-<num>  provisioned true
```

Comment:
I cannot comment on line 225, but there you have the old BMH name. Probably you don't need to check that the state is "provisioned" for the 2nd time.

Reply:
Updated as suggested.